templates/feedback.html

```diff
@@ -28,28 +28,82 @@ <h2>Here is your Today's News Recommendations.</h2>
 						<img src="{{image.url}}">
 					{% endif %}
 				</div>
-				<div class="column is-7">
-
-						<a href="{{ impression.article.url }}">{{ impression.headline }}</a>
-						<p>{{ impression.subhead }}</p>
+				<div class="column is-6">
+					<a href="{{ impression.article.url }}">{{ impression.headline }}</a>
+					<p>{{ impression.subhead }}</p>
 				</div>
-				<div class="column is-1">
-					<form action="{{ url_for('feedback') }}" method="POST">
-					<input type="hidden" id="newsletter_id" name="newsletter_id" value="{{ impression.newsletter_id }}">
-					<input type="hidden" id="impression_id" name="impression_id" value="{{ impression.impression_id }}">
-					<button type="submit" name="articlefeedbackType" value="positive"
-					class="emoji-button {% if impression.feedback == true %}highlight_green{% endif %}"
-					>
-						👍
-					</button>
-					<button type="submit" name="articlefeedbackType" value="negative"
-					class="emoji-button {% if impression.feedback == false %}highlight_red{% endif %}"
-					>
-						👎
-					</button>
-				</form>
+				<div class="column is-2">
+					<form class="feedback-form feedback-class" action="{{ url_for('feedback') }}" method="POST">
+						<input type="hidden" id="newsletter_id" name="newsletter_id" value="{{ impression.newsletter_id }}">
+						<input type="hidden" id="impression_id" name="impression_id" value="{{ impression.impression_id }}">
+
+						<button type="submit"
+						name="articlefeedbackType"
+						value="positive"
+						class="emoji-button {% if impression.feedback == true %}highlight_green{% endif %}"
+						title="Show more articles like this"
+						>
+							👍
+
+						</button>
+
+						<button type="submit"
+						name="articlefeedbackType"
+						value="negative"
+						class="emoji-button {% if impression.feedback == false %}highlight_red{% endif %}"
+						title="Don't show articles like this"
+						>
+							👎
+						</button>
+					</form>
 				</div>
 		</div>
 		{% endfor %}
 </div>
+
+
+<script>
```

Comment:
On [other pages](https://github.com/CCRI-POPROX/poprox-web/blob/69ca3a6997b70a74b184343c15867635006ee614/templates/topics.html#L11) we put the scripts inside the `header` block, which is probably a good idea here too

Reply:
agreed. I'm pretty sure the template files have a head block you can use to put your script in the header. You've already wrapped everything in `document.addEventListener ("DOMContentLoaded"...);` so it should be fine at the top or bottom.